testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/settings/AppSettings.kt

```diff
@@ -9,72 +10,24 @@ import com.rakuten.tech.mobile.miniapp.js.userinfo.Contact
 import com.rakuten.tech.mobile.miniapp.js.userinfo.Points
 import com.rakuten.tech.mobile.miniapp.js.userinfo.TokenData
 import com.rakuten.tech.mobile.miniapp.testapp.BuildConfig
-import com.rakuten.tech.mobile.testapp.BuildVariant
 import java.util.*
 
+@Suppress("TooManyFunctions")
 class AppSettings private constructor(context: Context) {
 
     private val manifestConfig = AppManifestConfig(context)
-    private val cache = Cache(context)
+    private val cache = Cache(context, manifestConfig)
```

Comment:
I think you dont need to pass `AppManifestConfig` here, you can set the values depending on settings page, `AppManifestConfig` is being used for sdk i guess,

Reply:
AppManifestConfig itself along with its AppSettings runs on the runtime even previously. 

Alright.